libfs/file.go

```diff
@@ -114,3 +118,65 @@ func (f *File) Close() error {
 	f.node = nil
 	return nil
 }
+
+func (f *File) getLockID() keybase1.LockID {
+	// We must not change or remove this since it'd break locking guarantees.
+	return keybase1.LockIDFromBytes(
+		bytes.Join([][]byte{
+			f.fs.GetLockNamespace().Bytes(),
+			[]byte(f.Name()),
```

Comment:
Isn't this supposed to be a hash, so we don't expose the filename to the server?

Reply:
Yup! `keybase1.LockIDFromBytes` hashes the passed in bytes and turns it into an integer (keybase1.LockID is an integer) to send to `mdserver`

Reply:
Oh oops, I forgot about that.  Sorry!

Reply:
NP!